akka-projection-cassandra/src/test/scala/akka/projection/cassandra/ContainerSessionProvider.scala

```diff
@@ -23,25 +24,25 @@ final class ContainerSessionProvider extends CqlSessionProvider {
 
   override def connect()(implicit ec: ExecutionContext): Future[CqlSession] = started.map { _ =>
     CqlSession.builder
-      .addContactEndPoint(
-        new DefaultEndPoint(
-          InetSocketAddress
-            .createUnresolved(
-              container.cassandraContainer.getContainerIpAddress,
-              container.cassandraContainer.getFirstMappedPort.intValue())))
+      .addContactEndPoint(new DefaultEndPoint(InetSocketAddress
+        .createUnresolved(container.getContainerIpAddress, container.getFirstMappedPort.intValue())))
       .withLocalDatacenter("datacenter1")
       .build()
   }
 }
 
 object ContainerSessionProvider {
   private val disabled = java.lang.Boolean.getBoolean("disable-cassandra-testcontainer")
-  private lazy val container: CassandraContainer = CassandraContainer()
+  private lazy val container = new CassandraContainer()
   lazy val started: Future[Unit] = {
     if (disabled)
       Future.successful(())
     else
-      Future.fromTry(Try(container.start()))
+      Future.fromTry(Try {
+        container.withStartupCheckStrategy(new IsRunningStartupCheckStrategy)
+        container.withStartupAttempts(5)
+        container.start()
```

Comment:
This is supposed to be a fluent API, but somehow the Scala compiler complains about it. Probably because of how those classes are encoded in Java, using a self-type param. 

Most strange is that I got it working on JDBC and Cassandra, but not compiling on Slick while the Slick one is using the same stuff as the JDBC.